### PR TITLE
fix(service): propagate original args into original inject fn

### PR DESCRIPTION
**Pull request by srowhani**

### CODE REVIEW

File: vendor/service-inject-3.js [hunk 1]
```diff
@@ -1,13 +1,21 @@
 /* globals Ember */
 /* eslint-disable ember/new-module-imports */
 
+const { isElementDescriptor } = Ember.__loader.require('@ember/-internals/metal');
```
Comment (rwjblue) [inline vendor/service-inject-3.js]: I don't think we can rely on this module + method existing (this is likely why CI is failing for older Ember versions)

**Comment by rwjblue** [comment]
Thanks for working on this!

File: vendor/service-inject-3.js (hunk 1 above)
Comment (srowhani) [reply vendor/service-inject-3.js]: I see! Added some logic in tests to only run if the version is greater than 3.10, and check to see if getElementDescriptor is present before invoking.

**Comment by elwayman02** [comment]
Is this ready to go or does it need more changes @rwjblue?

**Comment by rwjblue** [comment]
Thanks for the ping here @elwayman02!

**Comment by rwjblue** [comment]
Thank you @srowhani!

**Pull request merged**